src/core/net/ip6.hpp

```diff
@@ -434,13 +356,13 @@ class Ip6 : public InstanceLocator
     otError RemoveMplOption(Message &aMessage);
     otError HandleOptions(Message &aMessage, Header &aHeader, bool &aForward);
     otError HandlePayload(Message &aMessage, MessageInfo &aMessageInfo, uint8_t aIpProto);
-    int8_t  FindForwardInterfaceId(const MessageInfo &aMessageInfo);
+    bool    ShouldForwardToThread(const MessageInfo &aMessageInfo);
```

Comment:
Is this method supposed to be const? Are we missing a const getter in locate-getters. @abtink